.travis.yml

```diff
@@ -14,16 +14,21 @@ env:
     - REGISTRY_USER=praekeltorgdeploy
     - secure: "OIjqHeoNs0jHoqk5mAY2qux+qsokxcuwxBDDdbwCyJg0P5EQAu9XakIzThpZZixIJ/3VAATFdHK/xlNzJeanq7LUZr11DJKBD3H7jmYE2hm5+3qjjNh8KW8Z8NluezmwGWz6gkb8sCsgO2+k7sKC+4h4AERBOceF21V+iOUjBeCw1Q9DVDoBEznbIv1MJhe4G8V5Z3PEFaoerJ4Cy1VsEQRpHEzWyIBxgr4mXfGv4H6PLgTTNSs0+afSCJL5Nc5EU67NpLFTPBIS65yD5xDkLA+Tl4cLomJAwZNe8J9+GQWSua9ehqj0kPOuolkxGWZ5M/Z+yk80i9FNy0CdEWoEKrQMOCnnOwK769PWyKyzYCI9FWPky1rRAqFAdGmMspI3i1qm3wMUUERGqB9U0XuywPbrvqTVKqrdWP69Gwx84ZGvxnODN3nW1Yot/M75p9JbwbnBy+4tRDEqdXGbU1hefsCa7nA51CG1v8kybx6BmCRGWPUDTAR1m+ugRWvJjsaGOSILsSjUcQtlZbGSomLz0rJUgQi2qQEPKVbXv9zx8b9lzIYgxpKpLlONDgNBeFiW5RdT7yDvRUpDk0oSUi0YafkVqON6RuyQmmcg/dsXNtPr/712jqGO2BV3YPpiFB8xAZMWnfaBCe9dOSEDdo7f9PFvz0vUjFoeOC0FnVdd4DY="
   matrix:
-    # Tag 2.7 as 2, 3.6 as 3 and latest, and 3.5 as just 3.5...
+    # Tag 2.7 as 2, 3.7 as 3 and latest, and 3.6 as just 3.6...
     - PYTHON_VERSION=2.7 PYTHON_LATEST=  ALPINE_VERSION=3.6 ALPINE_LATEST=  SEMVER_PRECISION=1
-    - PYTHON_VERSION=2.7 PYTHON_LATEST=  ALPINE_VERSION=3.7 ALPINE_LATEST=1 SEMVER_PRECISION=1
-    - PYTHON_VERSION=3.6 PYTHON_LATEST=1 ALPINE_VERSION=3.6 ALPINE_LATEST=  SEMVER_PRECISION=1
-    - PYTHON_VERSION=3.6 PYTHON_LATEST=1 ALPINE_VERSION=3.7 ALPINE_LATEST=1 SEMVER_PRECISION=1
+    - PYTHON_VERSION=2.7 PYTHON_LATEST=  ALPINE_VERSION=3.7 ALPINE_LATEST=  SEMVER_PRECISION=1
+    - PYTHON_VERSION=2.7 PYTHON_LATEST=  ALPINE_VERSION=3.8 ALPINE_LATEST=1 SEMVER_PRECISION=1
+    - PYTHON_VERSION=3.6 PYTHON_LATEST=  ALPINE_VERSION=3.6 ALPINE_LATEST=  SEMVER_PRECISION=2
+    - PYTHON_VERSION=3.6 PYTHON_LATEST=  ALPINE_VERSION=3.7 ALPINE_LATEST=  SEMVER_PRECISION=2
+    - PYTHON_VERSION=3.6 PYTHON_LATEST=  ALPINE_VERSION=3.8 ALPINE_LATEST=1 SEMVER_PRECISION=2
+    - PYTHON_VERSION=3.7 PYTHON_LATEST=1 ALPINE_VERSION=3.8 ALPINE_LATEST=1 SEMVER_PRECISION=1
 
 install:
   - git clone https://github.com/docker-library/official-images.git ~/official-images
 
 before_script:
+  # tianon's hack to get GPG to work vaguely reliably
+  - wget -qO- 'https://github.com/tianon/pgp-happy-eyeballs/raw/master/hack-my-builds.sh' | bash
```

Comment:
I'm not enthusiastic about running arbitrary code from the public internet as part of our base image builds. Can we at least pin this to a particular version so we aren't open to attack-by-compromised-external-repo?

Reply:
Pinning is a good idea.